addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,351 @@
+            <channel id="batteryLevel" typeId="batterylevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <thing-type id="rfy">
+        <label>RFXCOM Rfy thing</label>
+        <description></description>
+
+        <channels>
+            <channel id="command" typeId="command-channel"/>
+            <channel id="shutter" typeId="shutter-channel"/>
+            <channel id="signalLevel" typeId="signallevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <thing-type id="security1">
+        <label>RFXCOM Security1 thing</label>
+        <description></description>
+
+        <channels>
+            <channel id="status" typeId="status-channel"/>
+            <channel id="contact" typeId="contact-channel"/>
+            <channel id="motion" typeId="motion-channel"/>
+            <channel id="signalLevel" typeId="signallevel-channel"/>
+            <channel id="batteryLevel" typeId="batterylevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <thing-type id="temperaturehumidity">
+        <label>RFXCOM Temperature-Humidity thing</label>
+        <description></description>
+
+        <channels>
+            <channel id="temperature" typeId="temperature-channel"/>
+            <channel id="humidity" typeId="humidity-channel"/>
+            <channel id="humidityStatus" typeId="humiditystatus-channel"/>
+            <channel id="signalLevel" typeId="signallevel-channel"/>
+            <channel id="batteryLevel" typeId="batterylevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <thing-type id="temperature">
+        <label>RFXCOM Temperature thing</label>
+        <description></description>
+
+        <channels>
+            <channel id="temperature" typeId="temperature-channel"/>
+            <channel id="signalLevel" typeId="signallevel-channel"/>
+            <channel id="batteryLevel" typeId="batterylevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <thing-type id="thermostat1">
+        <label>RFXCOM Temperature thing</label>
+        <description></description>
+
+        <channels>
+            <channel id="contact" typeId="contact-channel"/>
+            <channel id="setpoint" typeId="setpoint-channel"/>
+            <channel id="temperature" typeId="temperature-channel"/>
+            <channel id="signalLevel" typeId="signallevel-channel"/>
+            <channel id="batteryLevel" typeId="batterylevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <thing-type id="wind">
+        <label>RFXCOM Wind thing</label>
+        <description></description>
+
+        <channels>
+            <channel id="windSpeed" typeId="windspeed-channel"/>
+            <channel id="windDirection" typeId="winddirection-channel"/>
+            <channel id="signalLevel" typeId="signallevel-channel"/>
+            <channel id="batteryLevel" typeId="batterylevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <channel-type id="command-channel">
```

Comment:
I would remove "-channel" from all the channel type definitions as this is superfluous.